### PR TITLE
Create lobster-bisque.md

### PR DESCRIPTION
**Pull request by yiusa**

### CODE REVIEW

File: src/lobster-bisque.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+9. Deglaze the pan with the white wine and once the wine has dried up a bit add the fish stock.
+10. Let it simmer on low heat for 30 minutes, while the soup is boiling take the lobster meat and dice it into bitesized pieces.
+11. After the soup is done boiling run it through a fine sieve.
+12. Bring the strained soup back to  a light boil and add the diced lobster pieces.
+13. Let the soup boil for 4 more minutes to heat up the lobster and serve, finish of in the plate with a bite of heavy cream.
+
+## Contribution
+
+Yiusa
+-eth '0x68f1317c6512f0267fa711cafb6c134ae968fa80'
```
Comment (DGKSK8LIFE) [inline src/lobster-bisque.md:50]: Not sure if tagging your ETH wallet is allowed.

Comment (yiusa) [reply src/lobster-bisque.md:50]: I've tagged it on all my recipes i have posted here and they all got added before, if monero and btc are allowed i don't see why eth wouldn't be.

Comment (DGKSK8LIFE) [reply src/lobster-bisque.md:50]: Ok. Yeah I didn't read your other PRs closely

Comment (wesleydeal) [reply src/lobster-bisque.md:50]: BTC and XMR addresses are allowed, what's so different about ETH?

File: src/lobster-bisque.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+6. Add all diced vegetables and let it sauté for 2 minutes, add a dash of cognac and flambé, lower the heat to medium low.
+7. Rougly dice the tomatoes and add them together with the tomato puree, once again let it sauté for 2 minutes.
+8. Add the flour and give it all a stir, let it dry up in the pan.
+9. Deglaze the pan with the white wine and once the wine has dried up a bit add the fish stock.
+10. Tie up your herbs using kitchen twine and add them to your pot.
+11. Let it simmer on low heat for 30 minutes, while the soup is boiling take the lobster meat and dice it into bitesized pieces.
+12. After the soup is done boiling run it through a fine sieve.
+13. Bring the strained soup back to  a light boil and add the diced lobster pieces.
+14. Let the soup boil for 4 more minutes to heat up the lobster and serve, add pepper and salt to taste.
+15. Finish of in the plate with a bite of heavy cream.
```
Comment (hallzy) [inline src/lobster-bisque.md:45]:
```suggestion
15. Finish of in the plate with a bit of heavy cream.
```

File: src/lobster-bisque.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+- fresh parsley
+- heavy cream
+- canola oil
+- 2 tbsp butter
+
+## Directions
+
+1. Finely dice the onions, carrots, shallots and garlic and set aside for later.
+2. Blanche the lobster in boiling water for 2 minutes, remove from the water and let it leak.
+3. Cut it down the middle and remove the meat from the shell, set aside the meat for later.
```
Comment (hallzy) [inline src/lobster-bisque.md:33]:
```suggestion
3. Cut it down the middle and remove the meat from the shell, set the meat aside for later.
```

File: src/lobster-bisque.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+4. Place a stockpot on medium high heat and heat up enough oil to cover the bottom of the pan, add the lobster shells and cook them until they are bright red.
+5. Remove the oil from the pan and discard it, add your butter to the shells.
+6. Add all diced vegetables and let it sauté for 2 minutes, add a dash of cognac and flambé, lower the heat to medium low.
+7. Rougly dice the tomatoes and add them together with the tomato puree, once again let it sauté for 2 minutes.
+8. Add the flour and give it all a stir, let it dry up in the pan.
+9. Deglaze the pan with the white wine and once the wine has dried up a bit add the fish stock.
+10. Tie up your herbs using kitchen twine and add them to your pot.
+11. Let it simmer on low heat for 30 minutes, while the soup is boiling take the lobster meat and dice it into bitesized pieces.
```
Comment (hallzy) [inline src/lobster-bisque.md:41]:
```suggestion
11. Let it simmer on low heat for 30 minutes, while the soup is boiling take the lobster meat and dice it into bite-sized pieces.
```

File: src/lobster-bisque.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+12. After the soup is done boiling run it through a fine sieve.
+13. Bring the strained soup back to  a light boil and add the diced lobster pieces.
```
Comment (hallzy) [inline src/lobster-bisque.md:43]:
```suggestion
13. Bring the strained soup back to a light boil and add the diced lobster pieces.
```

File: src/lobster-bisque.md (hunk 2 above)
Comment (hallzy) [inline src/lobster-bisque.md:45]:
```suggestion
15. Finish off in the plate with a bite of heavy cream.
```

**Review by hallzy — CHANGES_REQUESTED** [review]
Do you have an image to go with this?

**Review by hallzy — APPROVED** [review]

File: src/lobster-bisque.md (hunk 2 above)
Comment (yiusa) [reply src/lobster-bisque.md]: Sadly i don't, its a recipe i don't cook often. I could look online for an image but it won't exactly match the recipe. Also im not 100% sure on how to add an image to a recipe in the first place. New to git, just like to cook and hate modern recipe websites.

Comment (hallzy) [reply src/lobster-bisque.md]: There is a rule against using images from google, so I wouldn't do that. All images here are taken by the creator or someone that used the recipe to make it.

For future, all you do is add an image file to the `data/pix` folder. Use the same name as the recipe file. If you have additional images for different steps of the recipe, then they will have the same name but will have numbers. example:

```
data/pix/lobster-bisque.webp
data/pix/lobster-bisque-01.webp
```

To add the image to the `.md` file you just add something like this, just look at how they are referenced in other recipe files.

Also, all images should be `webp` files, and less than 150kB

(A lot of this info is also on the main readme btw)

File: src/lobster-bisque.md (hunk 1 above)
Comment (LukeSmithxyz) [reply src/lobster-bisque.md:50]: Eth is obviously fine, but the format should be (in an unordered list):

```
- Yiusa, eth `0x68f1317c6512f0267fa711cafb6c134ae968fa80`
```

I've been going around and fixing yours to make them consistent with others.

**Pull request merged**